src/query/graphite/storage/m3_wrapper_test.go

```diff
@@ -26,6 +26,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/golang/mock/gomock"
```

Comment:
nit: Align the import with other extern imports?

Reply:
Damnit thought I managed to get them all right haha... 1 out of 118 not too bad :p